Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/util/security/VerificationKeysTest.kt

```diff
@@ -54,18 +66,21 @@ class VerificationKeysTest {
     }
 
     companion object {
+        private const val PUB_KEY =
```

Comment:
What do you think about moving the companion object to the beginning of the class? I personally like to see all consts/companions at first of the class so I know what consts are used in the code

Reply:
I like the opposite, I think the constants distract from the global variables and constructor arguments. 
Our goal is to follow the Kotlin code conventions which recommend companion objects at the end: 
https://kotlinlang.org/docs/reference/coding-conventions.html